migrations/20200831143314_endpointNameLength.js

```diff
@@ -0,0 +1,46 @@
+/*****
+ License
+ --------------
+ Copyright © 2017 Bill & Melinda Gates Foundation
+ The Mojaloop files are made available by the Bill & Melinda Gates Foundation under the Apache License, Version 2.0 (the "License") and you may not use these files except in compliance with the License. You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, the Mojaloop files are distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ * ModusBox
+ - Kevin Leyow <kevin.leyow@modusbox.com>
+ --------------
+ ******/
+
+'use strict'
+
+exports.up = async (knex) => {
+  return await knex.schema.hasTable('endpointType').then(function(exists) {
+    if (exists) {
+      return knex.schema.alterTable('endpointType', function(t) {
+        t.string('name', 100).alter();
```

Comment:
Is it fine for us to make this `VARCHAR` bigger or do we need to make our endpoint names smaller > - >

Reply:
maybe let keep it as is and shorten THIRDPARTY prefix to TP... ?

Reply:
and CALLBACK to CB

Reply:
@lewisdaly @sridharvoruganti  any opinion on this? length migration vs endpoint refactor?

Reply:
either is fine for me, but if we rename these endpoints then code changes required in central-services-shared and thirdparty-api-adapter.

Reply:
Yeah let's shorten the names here - migrations in central-ledger should be avoided if at all possible

Reply:
done. shortened the endpoint names Thirdparty -> TP. Callback -> CB. Authorizations -> Auth.

I'll make the changes in central-services-shared and thirdparty-api-adapter after this is merged.